File: Java/Booking/DiscountItem.java
package LTD_API;

public class DiscountItem
{
    private int orderItemId;
    private String name;
    private String code;
    private decimal price;
    
    public int getOrderItemId()
    {
        return orderItemId;
    }
    public void setOrderItemId(int orderItemId)
    {
        this.orderItemId = orderItemId;
    }

    public String getName()
    {
        return name;
    }
    public void setName(String name)
    {
        this.name = name;
    }

    public String getCode()
    {
        return code;
    }
    public void setCode(String code)
    {
        this.code = code;
    }

    public decimal getPrice()
    {
        return price;
    }
    public void setPrice(decimal price)
    {
        this.price = price;
    }
}

File: PHP/Booking/DiscountItem.php
<?php
namespace LTD_API;

class DiscountItem
{
    private $orderItemId;
    private $name;
    private $code;
    private $price;
    
    public function getOrderItemId()
    {
        return $this->orderItemId;
    }
    public function setOrderItemId($orderItemId)
    {
        $this->orderItemId = $orderItemId;
    }

    public function getName()
    {
        return $this->name;
    }
    public function setName($name)
    {
        $this->name = $name;
    }

    public function getCode()
    {
        return $this->code;
    }
    public function setCode($code)
    {
        $this->code = $code;
    }

    public function getPrice()
    {
        return $this->price;
    }
    public function setPrice($price)
    {
        $this->price = $price;
    }
}

